physlr/physlr.py

```diff
@@ -513,13 +548,13 @@ def physlr_cut_chimera(g):
             print()
 
     @staticmethod
-    def determine_backbones(g):
+    def determine_backbones(g, junction_threshold=0):
```

Comment:
```suggestion
    def determine_backbones(g, junction_threshold):
```
Remove the default parameter and update all callers.

Reply:
Where should I prefer using `Physlr.args.min_branch` and where should I instead prefer passing the parameter to the function @sjackman ? 
I removed the parameter from the functions `determine_backbones` and `determine_backbones_and_remove_chimera`, but I am still passing it (with no default) to other sub-functions.